### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by XhmikosR**
* use https when possible
* use rel=noopener for external links

/CC @LeaVerou

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,7 +54,7 @@ <h1>How to use</h1>
 <script src="contrast-ratio.js"></script>
 
 <script>_gaq = [['_setAccount', 'UA-117109922-1'], ['_trackPageview']];</script>
-<script src="http://www.google-analytics.com/ga.js" async></script>
+<script src="https://www.google-analytics.com/ga.js" async></script>
```
Comment (XhmikosR) [inline index.html]: BTW, maybe you should switch to the new GA snippet while at it

**Comment by LeaVerou** [comment]
Hey, thanks for contributing!

Slashes in self-closing tags are perfectly valid HTML5 (though optional). I like using them because they aid readability by signifying that a tag is self-closing without expecting the reader to memorize the list of self-closing tags.

Regarding https, I just merged #29 which fixed some of these. For the rest, if you could open a new PR that doesn’t remove my beloved slashes, I'd be delighted to merge it :)

Same applies for the new GA snippet: I don't have the time to do it right now, but would gladly merge a PR for it :)

**Comment by XhmikosR** [comment]
@LeaVerou: I could just rebase this instead of you closing it ;)

**Comment by LeaVerou** [comment]
Wasn't sure if you'd prefer to do that. Reopened!

**Comment by LeaVerou** [comment]
Also just noticed the username. Γειά σου συμπατριώτη! 👋

**Comment by XhmikosR** [comment]
Haha, Καλησπέρα :)

OK, I made the changes, bu the GA snippet isn't the latest one you will get if you visit your GA account. I can revert that too, or you can paste the snippet from your GA page :)

**Comment by LeaVerou** [comment]
Thanks, but the slashes are still showing as removed?

**Comment by XhmikosR** [comment]
Did you refresh the page? This is the raw patch I have pushed in this branch https://github.com/LeaVerou/contrast-ratio/commit/b7ce0cc4c0c053185ee62db9ae37cd022d963374.patch

**Comment by LeaVerou** [comment]
Maybe I hadn't, I do see it now! (sorry, τρέχω και δε φτάνω σήμερα)
Merged, thanks!

**Pull request merged**